Automattic/wp-calypso
# Pattern Library: Increase `viewportWidth` in list view

Fixes https://github.com/Automattic/dotcom-forge/issues/6550

## Proposed Changes

In the pattern library, pattern previews in the list view have a width of 1205px at most. We recently discussed whether we should increase the viewport width inside the preview (i.e., zoom out the previews slightly). I think this is a good idea. The list view is our "full-scale preview," but 1205px is quite a narrow width compared to most computer screens, so using a slightly wider viewport seems reasonable.

## Testing Instructions



1. Navigate to `/patterns/about`
2. Ensure that the pattern previews look decent
3. Ensure that resizing still works
